content/blog/survival-analysis-parsnip-adjacent/index.Rmd

```diff
@@ -0,0 +1,227 @@
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+library(tidymodels)
+library(censored)
+```
+
+
+Survival analysis is an important field in modelling and there are many R packages available which implement various models, from "classic" parametric models to boosted trees. While they cover a great variety of model types, they also come with considerable amounts of heterogeneity in syntax and levels of documentation. The tidymodels framework is a collection of R packages for modeling and machine learning using tidyverse principles. It provides a consistent interface to a variety of modelling functions along with tools for resampling, assessing performance, and hyperparameter tuning.
+
+In terms of survival analysis, however, tidymodels so far hasn't been as feature-rich as we'd like it to be, e.g., the available models only covered parametric survival models. With this post we'd like to provide an update on recent work to extend the tooling for survival analysis in tidymodels and gather feedback on future plans.
+
+Making survival analysis a first-class citizen in tidymodels requires touching several aspects across the collection of packages:
+
+- Pre-processing which accommodates the data structure of censored data.
+- A bigger selection of models which can be specified, fitted, and used for prediction.
+- Additional performance metrics tailored to survival data.
+
+Our focus so far has been on model specification, fitting, and prediction so let's dive in and see what's new!
+
+
+### Model specification
+
+Several engines can be used for multiple purposes with the same model type, e.g. the `rpart` engine for `decision_tree()` can be used for classification or regression. This distinction can be made in parsnip by specifying the mode of a model.
+We have now introduced a new mode `"censored regression"` in parsnip for models which can be used for survival analysis. The aforementioned `decision_tree()` with the `rpart` engine can also be used to fit a survival decision tree:
+
+```{r new-mode}
+library(censored)
+
+decision_tree() %>% 
+  set_engine("rpart") %>% 
+  set_mode("censored regression")
+
+```
+
+You may notice a new package name here, `censored`. The number of models supported in tidymodels is growing. The core models live in parsnip, however there are several parsnip-adjacent extension packages with additional models such as [discrim](https://discrim.tidymodels.org/) for discriminant analysis models, [poissonreg](https://poissonreg.tidymodels.org/) for Poisson regression models, and [baguette](https://baguette.tidymodels.org/) for ensemble models via bagging. The [censored](https://tidymodels.github.io/censored/) package is the extension package for censored regression. 
```

Comment:
Should this paragraph be restructured to move the link to {censored} further up?